plutus-tx-plugin/src/Language/PlutusTx/Compiler/Builtins.hs

```diff
@@ -281,39 +277,39 @@ defineBuiltinTerms = do
 
     -- Integer builtins
     do
-        let term = mkStaticBuiltin PLC.AddInteger
+        let term = mkBuiltin PLC.AddInteger
         defineBuiltinTerm 'Builtins.addInteger term [int]
     do
-        let term = mkStaticBuiltin PLC.SubtractInteger
+        let term = mkBuiltin PLC.SubtractInteger
         defineBuiltinTerm 'Builtins.subtractInteger term [int]
     do
-        let term = mkStaticBuiltin PLC.MultiplyInteger
+        let term = mkBuiltin PLC.MultiplyInteger
         defineBuiltinTerm 'Builtins.multiplyInteger term [int]
     do
-        let term = mkStaticBuiltin PLC.DivideInteger
+        let term = mkBuiltin PLC.DivideInteger
         defineBuiltinTerm 'Builtins.divideInteger term [int]
     do
-        let term = mkStaticBuiltin PLC.RemainderInteger
+        let term = mkBuiltin PLC.RemainderInteger
         defineBuiltinTerm 'Builtins.remainderInteger term [int]
     do
-        term <- wrapRel intTy 2 $ mkStaticBuiltin PLC.GreaterThanInteger
+        term <- wrapRel intTy 2 $ mkBuiltin PLC.GreaterThanInteger
         defineBuiltinTerm 'Builtins.greaterThanInteger term [int, bool]
     do
-        term <- wrapRel intTy 2 $ mkStaticBuiltin PLC.GreaterThanEqInteger
+        term <- wrapRel intTy 2 $ mkBuiltin PLC.GreaterThanEqInteger
         defineBuiltinTerm 'Builtins.greaterThanEqInteger term [int, bool]
     do
-        term <- wrapRel intTy 2 $ mkStaticBuiltin PLC.LessThanInteger
+        term <- wrapRel intTy 2 $ mkBuiltin PLC.LessThanInteger
         defineBuiltinTerm 'Builtins.lessThanInteger term [int, bool]
     do
-        term <- wrapRel intTy 2 $ mkStaticBuiltin PLC.LessThanEqInteger
+        term <- wrapRel intTy 2 $ mkBuiltin PLC.LessThanEqInteger
         defineBuiltinTerm 'Builtins.lessThanEqInteger term [int, bool]
     do
-        term <- wrapRel intTy 2 $ mkStaticBuiltin PLC.EqInteger
+        term <- wrapRel intTy 2 $ mkBuiltin PLC.EqInteger
```

Comment:
@michaelpj we really should derive all of those via type schemes, why do that manually.

Reply:
I was too lazy to do it automatically :D